homeassistant/components/media_player/__init__.py

```diff
@@ -540,6 +564,11 @@ def source_list(self):
         return None
 
     @property
+    def is_shuffling(self):
```

Comment:
I am confused, this should not work. You define the attribute as 'shuffle' above and so that should be the name.